### PR TITLE
Render templates in tests using TemplateService

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

**Review by MikelAlejoBR — APPROVED** [review]
LGTM!

File: engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java [hunk 1]
```diff
@@ -138,25 +135,20 @@ protected String generateAggregatedEmailBody(Action action) {
...
+    @Deprecated
+    protected String generateEmail(TemplateInstance template, Object actionOrEvent) {
 
+        String result = templateService.renderTemplate(createUser(), actionOrEvent, template);
         writeOrSendEmailTemplate(result, template.getTemplate().getId() + ".html");
 
         return result;
     }
 
+    @Deprecated
```
Comment (g-duval) [inline engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java:147]: Both `generateEmail` methods are now tagged as `Deprecated`.
The new equivalent method isn't it missing ?

Comment (josejulio) [reply engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java:147]: I guess that's wrong. These shouldn't be deprecated, thanks!

**Review by g-duval — APPROVED** [review]
LGTM

**Pull request merged**